### PR TITLE
komga: Add version 0.157.0

### LINKED ISSUE #8994: [Request] Komga
### Criteria

For a package to be acceptable in the Extras bucket, it should be:

- [x] a well-known and widely used tool (e.g. if it's a GitHub project, it should have at least 100 stars and 50 forks)
- [x] the latest stable version of the program

### Information

**Name:** Komga

**Description:** Media server for comics/mangas/BDs with API and OPDS support

**Homepage:** https://komga.org/

**Download link(s):** https://github.com/gotson/komga/releases/

**Some indication of popularity/repute:**
[Readme Card](https://github.com/gotson/komga)

### PR DESCRIPTION
**Pull request by LazyGeniusMan**
Add Komga, a media server for comics/mangas/BDs with API and OPDS support.

Closes #8994

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by LazyGeniusMan** [comment]
It's working fine in my local test. Is it fail because of `scripts` files not available yet in the repo?

File: bucket/komga.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+    "version": "0.157.0",
+    "description": "Media server for comics/mangas/BDs with API and OPDS support",
+    "homepage": "https://komga.org/",
+    "license": {
+        "identifier": "MIT",
+        "url": "https://github.com/gotson/komga/blob/master/LICENSE"
+    },
+    "suggest": {
+        "JDK": "java/temurin-lts-jdk"
```
Comment (niheaven) [inline bucket/komga.json:10]:
```suggestion
        "JDK": [
            "java/oraclejdk",
            "java/openjdk"
        ]
```

File: bucket/komga.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    },
+    "notes": [
+        "Default URL is http://localhost:8080",
+        "Create application.yml inside config folder to change configuration",
+        "More information: https://komga.org/installation/configuration.html"
+    ],
```
Comment (niheaven) [inline bucket/komga.json:16]:
```suggestion
    "notes": "Default URL is http://localhost:8080",
```

File: bucket/komga.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    "url": [
+        "https://github.com/gotson/komga/releases/download/v0.157.0/komga-0.157.0.jar#/komga.jar",
+        "https://github.com/ScoopInstaller/Extra/raw/master/scripts/komga/komga.ps1",
+        "https://github.com/ScoopInstaller/Extra/raw/master/scripts/komga/komga-log.ps1"
+    ],
+    "hash": [
+        "3e440beb57caa65b7a8758dc14220c111f997f3a300cfe3ae9d47501a0c7c394",
+        "5ffb11d99bb6f38f403639316fe4cc009fb31b0cd70cf229522f4f7b197cdd22",
+        "879cce4d614ae5e92dcbd6133d3173edc64f41ed1ee900d7d00f2a579c79d9ec"
+    ],
```
Comment (niheaven) [inline bucket/komga.json:26]:
```suggestion
    "url": "https://github.com/gotson/komga/releases/download/v0.157.0/komga-0.157.0.jar#/komga.jar",
    "hash": "3e440beb57caa65b7a8758dc14220c111f997f3a300cfe3ae9d47501a0c7c394",
```

File: bucket/komga.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    "bin": [
+        "komga.ps1",
+        "komga-log.ps1"
```
Comment (niheaven) [inline bucket/komga.json:29]:
```suggestion
        [
            "komga.jar",
            "komga",
            "--komga.config-dir=\"$dir\\config\""
        ]
```

File: bucket/komga.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    ],
+    "persist": "config",
+    "checkver": {
+        "github": "https://github.com/gotson/komga",
+        "regex": "komga-([\\d.]+)\\.jar"
```
Comment (niheaven) [inline bucket/komga.json:34]:
```suggestion
        "github": "https://github.com/gotson/komga"
```

File: bucket/komga.json [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    },
+    "autoupdate": {
+        "url": [
+            "https://github.com/gotson/komga/releases/download/v$version/komga-$version.jar#/komga.jar",
+            "https://github.com/ScoopInstaller/Extra/raw/master/scripts/komga/komga.ps1",
+            "https://github.com/ScoopInstaller/Extra/raw/master/scripts/komga/komga-log.ps1"
+        ]
```
Comment (niheaven) [inline bucket/komga.json:41]:
```suggestion
        "url": "https://github.com/gotson/komga/releases/download/v$version/komga-$version.jar#/komga.jar",
        "hash": {
            "url": "$url.sha256"
        }
```

**Review by niheaven — CHANGES_REQUESTED** [review]
Also delete two scripts

**Pull request merged**